### DISCUSSION TITLE
motion/variables/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#324] [Website Discussions]
# motion/variables/

On this page is a list of tokens you can use in Apple Motion to display metadata in Motion Templates, that will also appear in Final Cut Pro.

https://fcp.cafe/motion/variables/

**Comment by mkuhnkey** [comment]
Motion Topic:
https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/motion_XML_guide/Overview/Overview.html#//apple_ref/doc/uid/TP40007455-CH4-SW4

May be of some interest....a little off topic?

**Reply by latenitefilms** [reply]
Definitely of interest! I forgot this document even existed!

**Reply by latenitefilms** [reply]
Added here: https://fcp.cafe/motion/xml/
